### DISCUSSION TITLE
How to display my project images on README

### DISCUSSION THREAD

**Discussion by Xbener** [community/community#133587] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Everyone, I completed my own small project. I am going to display images to introduce my project on README.
I had tried several times, but not yet.
Plz teach me how to display images.

**Comment by nexy39** [comment, marked as answer]
oh, nice discussion. Plz follow steps.
1.Open the Markdown file (: README.md) in your project where you want to display the image.
2.Add the Markdown syntax for embedding an image. Use the following format:
![Alt text](images urls)
`![Alt text]: This is the alt text that describes the image. It helps with accessibility and SEO. Replace Alt text with a description of your image.`
`(URL): This is the URL of the image you want to display. If your images are in assets folder of your repo, it’s `
_(assets/project_images.png/jpg)_
Thanks

**Reply by Xbener** [reply]
good answer. _All_ images were displayed.

**Reply by nexy39** [reply]
what do u mean?
